src/app/wizard/services/app-launcher-dependency-check.service.ts

```diff
@@ -18,7 +20,7 @@ export class AppLauncherDependencyCheckService implements DependencyCheckService
       projectName: undefined,
       projectVersion: '1.0.0-SNAPSHOT',
       spacePath: undefined,
-      targetEnvironment: undefined
+      targetEnvironment: this.config.get('targetenvironment_skip') === 'true' ? 'os' : undefined
```

Comment:
@edewit could we make it a boolean in the config?

Reply:
@edewit is first cluster still auto selected when hidden?

Reply:
signature of config is:
```
Config.get(key: string): string
```

Reply:
@edewit ok then no other choice..  It could be good to add a `Config.getBoolean` in `ngx-launcher` in the future (but let's wait for the time we have another boolean).

Reply:
There was also a plan to have feature flags or something like that...